cloudify_gcp/compute/tests/test_instance.py

```diff
@@ -38,15 +38,18 @@ def test__get_script_string(self):
             'type': 'string',
             'script': '📜',
             })
-
-        self.assertEqual('📜', response)
+        new_script_format = {
+            'key': 'startup-script',
+            'value': '📜'
+        }
+        self.assertEqual(new_script_format, response)
 
     def test__get_script_bare_string(self):
-        self.assertEqual('🐻', instance._get_script('🐻'))
```

Comment:
The new code appears to still support this, so why are we removing this test? (also if I'm mistaken and it doesn't then this is a breaking change and we should be releasing this as 2.0 not 1.0.2)

Reply:
OK. I have added a test and renamed the tests to check that both the new input format and the old input format are supported.

Is it a breaking change if the old format is still supported?

Reply:
@funkyHat